bin/stop_json_api.cmd

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
TBH this script seems like overkill to me. Can just do `docker compose down`. Which is why I never created an equivalent for stargate repo.

Reply:
I see your point. I'll remove the stop scripts with appropriate update on the README instructions then. Let me repurpose this PR to just fix the Start script issue and the related github workflow issue.